protocol/scripts/markets/launch_markets.py

```
@@ -0,0 +1,285 @@
+        "dydxprotocold",
+        "query",
+        "marketmap", 
+        "market-map",
+        "--node=" + node,
+        "--chain-id=" + chain
+    ]
+    
+    with open(f"{temp_dir}/staging-marketmap.yml", "w") as f:
+        subprocess.run(cmd, stdout=f)  
+
+def markets_to_add(temp_dir: str):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)